### ISSUE TITLE
Fine-tuned/End-to-end Result!

### ISSUE THREAD

**Issue by KORIYN** [hbdat/cvpr20_DAZLE#5]
Hi Hbdat,

Have you conducted an end-to-end/fine-tuned experiment? I conducted such an experiment, but the result I got was about H0.5, do you know the reason? Thanks!

Best wishes,
Gerong

**Comment by hbdat** [comment]
Hi there,

Thanks for reaching out.

Regarding the training of DAZLE, it turns out that self-calibration loss, used in our CVPR20 paper, is not ideal for end-to-end training since this increases unseen class scores on seen class samples.
Thus, backpropagating gradient from this loss through many neural-net layers would simply amplify this undesirable effect.

This motivates our latest work to generate unseen class features for training DAZLE end-to-end.
In this work, which is accepted at NeurIPS this year, we manage to train both the backbone network (ResNet101) and DAZLE to achieve 63.9 Harmonic mean on CUB dataset.
We plan to release the code of this paper soon under the following Github repository:
[https://github.com/hbdat/neurIPS20_CompositionZSL](https://github.com/hbdat/neurIPS20_CompositionZSL
)

Best,

Dat
